src/connectors/playirish.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+Connector.playerSelector = '.qt-musicplayer';
+
+Connector.artistTrackSelector = '#streamCurrentlyPlaying';
+
+Connector.getArtistTrack = () => {
+	let text = $(Connector.artistTrackSelector).text().substring(16);
```

Comment:
I thinks it's better to replace `NOW PLAYING:` string directly, as it's more clear.

Reply:
Noted, added that change